audio/data.py

```diff
@@ -299,10 +299,9 @@ def open(self, item) -> AudioItem:
             else:
                 mel = MelSpectrogram(**(cfg.sg_cfg.mel_args()))(sig)
                 if cfg.sg_cfg.to_db_scale: mel = SpectrogramToDB(top_db=cfg.sg_cfg.top_db)(mel)
-            mel = mel.squeeze().permute(1, 0).flip(0)
+            mel = mel.permute(0, 2, 1).flip(1)
```

Comment:
I think this should be 0,1,2 which would mean we could just drop permute here.  

On my testing, this is how it was at least.

Reply:
Recleared cache, sg is still horizontal. I may have changed stuff elsewhere that affected it and made it differ from yours. But checking this closely did help me to catch and remove `.flip()` as somehow that is causing spectros to be upside down again.

Reply:
The only reason I can think of why youd be experiencing that is the underlying implementation of `MelSpectrogram` in torchaudio may have changed